ui/Screen/Project/Source.svelte

```diff
@@ -102,22 +115,28 @@
   <ActionBar>
     <div slot="left">
       <div style="display: flex">
-        <div class="revision-selector-wrapper">
-          <RevisionSelector
-            loading={selectedRevision.request !== null}
-            on:select={onSelectRevision}
-            selected={selectedRevision.selected}
-            defaultBranch={project.metadata.defaultBranch}
-            {revisions} />
-        </div>
+        {#if !patchesTabSelected}
```

Comment:
This might be a UI/UX follow up, but it feels a bit weird to me that the tab menu bar shifts to the left when the branch selector is gone. 🤷
 
What do you think @juliendonck and @brandonhaslegs?

https://user-images.githubusercontent.com/158411/117034660-d382bd80-ad03-11eb-8557-fb7132c90d1a.mov

Reply:
I agree, but the only way around this would be to move it to the header. Which is something we discussed but didn't want to do.